splunk/common-files/Dockerfile

```diff
@@ -60,6 +60,8 @@ RUN groupadd -r -g ${GID} ${SPLUNK_GROUP} \
     && chmod 755 /sbin/updateetc.sh
 
 COPY --from=package --chown=splunk:splunk /minimal /opt
+RUN mkdir -p ${SPLUNK_HOME}/share/splunk/search_mrsparkle/modules.new \
```

Comment:
I think it would be better to make these changes in the "package" stage so that it doesn't require an additional image layer.

Reply:
ah, right. done.